lib/logstasher/railtie.rb

```diff
@@ -41,6 +41,20 @@ class Railtie < Rails::Railtie
     end
   end
 
+  def default_source
+    case RUBY_PLATFORM
+    when /darwin/
+      # NOTE: MacOS Sierra and later are setting `.local`
```

Comment:
The whole sentence is relevant. SIerra and later have the problem with unresolvable hostnames, based on users reporting problems.